.github/workflows/build_and_run_tests.yaml

```diff
@@ -21,5 +21,17 @@ jobs:
           toolchain: stable
       - name: Free up disk space
         run: rm -rf /opt/hostedtoolcache
-      - name: test
-        run: cargo test -- --skip integration
+      - name: Test
+        run: cargo test -- --skip integration --skip solidity_abi::tests::test_gas_costs
+
+  gas_costs_test:
```

Comment:
As discussed on the call, we won't add this new job yet. Only skip the flaky test for now and I'll work on making it better this week.